drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -185,17 +185,67 @@ matterManufacturer:
     vendorId: 0x1339
     productId: 0x006C
     deviceProfileName: light-color-level-2000K-7000K
+  - id: "4921/64"
+    deviceLabel: Cync Indoor Plug
+    vendorId: 0x1339
+    productId: 0x0040
+    deviceProfileName: plug-binary
 #Ledvance
+  - id: 4489/843
```

Comment:
There should be quotes around this

Reply:
Not a big deal, but in the future let's just make sure to check for this before approving. This will have an id that is an evaluation of this number (4489/843 = 5.3250...) as opposed to the value `4489/848`

Reply:
Fix: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1738